ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/fullyprivate/internal-registry/deprovision/cucushift-installer-rehearse-azure-ipi-fullyprivate-internal-registry-deprovision-chain.yaml

```diff
@@ -0,0 +1,6 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-fullyprivate-internal-registry-deprovision
```

Comment:
Seem like you can directly reference `cucushift-installer-rehearse-azure-ipi-fullyprivate-deprovision` chain in the install workflow, does not need to introduce this dedicated provision chain.

Reply:
Sorry, update late, just computer meets black screen before